### PR TITLE
Move files

### PR DESCRIPTION
**Pull request by jd-lara**
This PR moves the .jl files for testing to PowerSystems since we won't support .jl data files anymore.

### CODE REVIEW

**Comment by daniel-thom** [comment]
One general suggestion/comment.  In InfrastructureSystems I instituted a rule that says all test files must start with "test_".  The runtests.jl code skips any file that doesn't match.  We could do the same here and you could keep these jl files in /test.

File: test/test_aux_files/data_14bus_pu.jl [hunk 1]
```diff
@@ -0,0 +1,210 @@
+using PowerSystems
```
Comment (daniel-thom) [inline test/test_aux_files/data_14bus_pu.jl]: This can be deleted.

File: test/test_aux_files/data_5bus_pu.jl [hunk 2]
```diff
@@ -0,0 +1,243 @@
+using PowerSystems
```
Comment (daniel-thom) [inline test/test_aux_files/data_5bus_pu.jl]: ditto

**Comment by jd-lara** [comment]
@daniel-thom I renamed the files but prefer to keep them as aux files so we can call them for other tests later if needed.

**Pull request merged**